src/xinterpreter.cpp

```diff
@@ -67,7 +81,15 @@ SEXP try_parse(const std::string& code, int execution_counter) {
     interpreter::interpreter(int argc, char* argv[])
     {
         Rf_initEmbeddedR(argc, argv);
+
+        R_Outputfile = NULL;
+        R_Consolefile = NULL;
+
+        ptr_R_WriteConsole = nullptr;
+        ptr_R_WriteConsoleEx = WriteConsoleEx;
+        
         xeus::register_interpreter(this);
+        p_interpreter = this;
```

Comment:
This essentially considers there is only one `xeus_r::interpreter` instance. Is that ok ?